storm-core/test/clj/org/apache/storm/cluster_test.clj

```diff
@@ -308,11 +309,11 @@
 
 (deftest test-cluster-state-default-acls
   (testing "The default ACLs are empty."
-    (stubbing [zk/mkdirs nil
+    (stubbing [Zookeeper/mkdirs nil
```

Comment:
```
    (with-open [_ (proxy [MockedZookeeper] []
                    (mkdirsImpl [] nil))]
      (stubbing [zk/mk-client (reify CuratorFramework (^void close [this] nil))]
      (mk-distributed-cluster-state {})
    ;  (verify-call-times-for mkdirsImpl 1)  //help
     ; (verify-first-call-args-for-indices mkdirsImpl [2] nil)))  //help
```
@revans2  can u help me? I update the code, but I don't how  update "verify-call-times-for " and verify-first-call-args-for-indices here.